packages/monaco-graphql/package.json

```diff
@@ -36,6 +36,7 @@
   },
   "peerDependencies": {
     "graphql": "^15.5.0 || ^16.0.0",
-    "monaco-editor": "^0.20.0"
+    "monaco-editor": "^0.20.0",
+    "prettier": "^2.8.0 || ^3.0.0"
```

Comment:
added `prettier` as peerDeps since `prettier` is imported here https://github.com/graphql/graphiql/blob/6c3ca902d769893a4201f8856299af18b7e21a1b/packages/monaco-graphql/src/GraphQLWorker.ts#L141-L142